tutorials/raw/plot_events.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+"""
+Working with events
+===================
+
+This tutorial describes event representation and how event arrays are used to
+subselect data.
+
+.. contents:: Page contents
+   :local:
+   :depth: 1
+
+As usual we'll start by importing the modules we need, loading some example
+data, and cropping it to save memory:
+"""
+
+import os
+import numpy as np
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+raw.crop(tmax=60).load_data()
+
+###############################################################################
+# The tutorial :doc:`../intro/plot_object_annotations` describes in detail how
+# to obtain an Events array from either :term:`STIM channels <stim channel>` or
+# embedded event arrays within a raw data file, so we won't discuss that in
+# detail here. As a quick reminder, the :ref:`sample dataset <sample-dataset>`
+# includes experimental events recorded on channel ``STI 014``. Here parse the
+# events from that channel using :func:`mne.find_events`:
```

Comment:
I would put the first part of this tutorial at the top. 

`This tutorial describes ..` `What this tutorial do not discuss is ...` `This is cover in detail in :doc:../intro/plot_object_annotatios`.

If I had to put the reminder I would also put it in the top, but to make it programmatically we need the first snipped loading the python stuff. So I would either put it as an inline comment that links to this line in the other tutorial or drop it altogether and assume that the link is enough.
ie:
```md
As a quick reminder, [`events = mne.find_events(raw, stim_channel='STI 014')`](https://13302-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/intro/plot_object_annotations.html#converting-a-stim-channel-signal-to-an-events-array) is used to extracts events in :ref:sample dataset<bla> since they are recorded on channel STI014.
```

Reply:
It's not a reminder. This is how you obtain events from a raw object. I would give it a title and a little comment. 

```rst
###############################################################################
# Obtaining events from a raw object
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#
# It the raw object contains stim channel (as is the case of :ref:`sample dataset <sample-dataset>` which includes experimental events recorded on channel ``STI 014``) the events can be obtained using :func:`mne.find_events`:

events = mne.find_events(raw, stim_channel='STI 014')
```

Reply:
see my [outline proposal](https://github.com/mne-tools/mne-python/pull/6358/files#r286413570)

Reply:
cross reference with https://13302-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/intro/plot_introduction.html#detecting-experimental-events

Reply:
you're right that this section needed some smoothing. I'm -1 on adding a subsection title though, because this section only *carries out* the action without teaching how to do it.  That's the reason for the cross-ref to the other tutorial, where the teaching happens.  Here's what I've come up with:

```
# The tutorial :doc:`../intro/plot_object_annotations` describes in detail the
# different ways of obtaining an :term:`Events array <events>` from a
# :class:`~mne.io.Raw` object (see the section
# :ref:`detecting-experimental-events` for details). Since the :ref:`sample
# dataset <sample-dataset>` includes experimental events recorded on
# :term:`STIM channel` ``STI 014``, we'll start this tutorial by parsing the
# events from that channel using :func:`mne.find_events`:
```